__init__.py

```diff
@@ -11,17 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import re
+
 import requests
 from datetime import timedelta
 
-from adapt.intent import IntentBuilder
+import re
```

Comment:
`re` is builtin and `requests` is third party.  these two imports need to be flipped